src/components/tabs/js/tabsController.js

```diff
@@ -764,15 +765,58 @@ function MdTabsController ($scope, $element, $window, $mdConstant, $mdTabInkRipp
   function refreshIndex () {
     ctrl.selectedIndex = getNearestSafeIndex(ctrl.selectedIndex);
     ctrl.focusIndex    = getNearestSafeIndex(ctrl.focusIndex);
+  } 
+
+  /**
+   * Rapid fires function to compensate for rendering lag for post-render functions.
+   * Note: Only use on inexpensive functions to avoid performace issues.
+   * @returns {*}
+   */
+  function cascadeTimeouts (func) {
+    var timeouts = [],
+        offsets = [50, 100, 300, 1000];
+
+    for (var i = 0; i < offsets.length; i++) {
+      timeouts.push($timeout(func, offsets[i]));
+    }
+
+    ctrl.scope.$on('$destroy', function () {
+      for (var i = 0; i < timeouts.length; i++) {
+        $timeout.cancel(timeouts[i]);
+      }
+    });   
+  }
+
+  /**
+   * Updates element's height of fill available space of parent element.
+   * @returns {*}
+   */
+  function updateHeightFromParent () {
+    var parent = $element[0].parentNode,
+        parentHeight = parent.clientHeight,
+        siblingsHeight = 0;
+
+    angular.forEach(parent.children, function(child){
```

Comment:
Missing space `function(child){` -> `function(child) {`

Reply:
@Splaktar to save another needless delay, I ignored your comment and changed it to `function (child) {`. Hope that's cool.

Reply:
That kind of attitude and feedback is not productive or appreciated.